api/migrations/0002_data_migration.py

```diff
@@ -10,6 +10,7 @@
     STUDENT_AGE_RANGES,
     STUDENT_AGE_RANGES_FOR_MATCHING,
     STUDENT_AGE_RANGES_FOR_TEACHER, LanguageLevelId,
+    TIME_SLOTS
```

Comment:
this doesn't seem to be related to Django version, though. Maybe it will disappear after merging `master` into this branch?

Reply:
This is my changes